tests/unit/compile/test_serialization.py

```diff
@@ -107,11 +108,18 @@ def __init__(self):
 
 class BasicStateful(Component):
     def __init__(self):
-        self.x = 2019
+        self.x = random.randint(0, 100000)
```

Comment:
lets not have flaky tests here. Just use a hardcoded random number.

Reply:
The number needs to be different for each initialization. We could instead have the number passed in and use an incremented variable defined in each individual test case.

Reply:
Whatever it takes to avoid `random.randint` !!